Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:2
+
+RUN set -x \
+    && apt-get update -qq \
+    && apt-get install -y -qq --no-install-recommends gcc git musl-dev python-dnspython python-xdg \
```

Comment:
here in the debian package, the build-deps are:

```
Build-Depends: debhelper (>= 9),
               gettext,
               python-all-dev (>= 2.7),
               python-requests (>= 2.2.1),
               qt4-dev-tools
```

not sure why you need musl/gcc here?

Reply:
Fixed... Sorry - my bad.